config/locales/service_providers.en.yml

```diff
@@ -13,7 +13,7 @@ en:
     aal_option_3: "Phishing-resistant MFA (e.g. WebAuthn or PIV/CAC cards) required, remember device disallowed (AAL2)"
     identity_assurance_level: "<p>Determines whether or not an application is permitted to request and receive verified attributes from users who have gone through Identity Verification.</p><p>Authentication only means your application will not be able to request or receive verified <a href='https://developers.login.gov/attributes/' target='_blank'>attributes</a>.</p><p>Note that Login.gov identity proofing services do not meet NIST IAL2 standards at this time. We continue to work toward achieving certification of compliance with the IAL2 standard from a third-party assessment organization.</p>"
     change_issuer: "The issuer cannot be changed, but you can create a new test app with a different issuer."
-    issuer: "This is a unique string to identify the app in the IdP, and is also used as the <code>client_id</code>. We recommend something like the following, replacing <code>agency_name</code> and <code>app_name</code> with your own.<br><i>For OpenID Connect: </i><code class='ml2'>urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name</code><br><i>For SAML: </i><code class='ml2'>urn:gov:gsa:SAML:2.0.profiles:sp:sso:agency_name:app_name</code>"
+    issuer: "<p>This is a unique string to identify the app in the IdP, and is also used as the <code>client_id</code>. We recommend something like the following, replacing <code>agency_name</code> and <code>app_name</code> with your own.</p><i>For OpenID Connect: <div></i><code class='ml2'>urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name</code><br><i></div>For SAML: <div></i><code class='ml2'>urn:gov:gsa:SAML:2.0.profiles:sp:sso:agency_name:app_name</code></div>"
```

Comment:
Can we remove the divs here? I don't think they would be serving a stylistic purpose and they are also in weird spots.

Reply:
they are serving a purpose. it is putting the issuer line on its own line, which allows a user to double click on `urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name` to more easily copy/paste it.

Reply:
if there is a better way to do this, i am happy to change it!

Reply:
The <p></p> tags I think would make more sense, or you could try <br /> they would make more sense here than <div> - but there is a possibility the <p> tag might add more space than desired.

Reply:
it does add a bit more space, but i think that's fine -- this seems like something we'd like to make more readable
<img width="592" alt="issuer description with more whitespace" src="https://github.com/18F/identity-dashboard/assets/5473830/e484e6c4-f6fe-41bb-b310-6f0682875601">

Reply:
code updated in https://github.com/18F/identity-dashboard/pull/671/commits/7fdeacdd20f3d19193be27731f6e2a05c6826063